upload-notebooks/README.md

```diff
@@ -0,0 +1,37 @@
+# Upload notebooks to TileDB Cloud
+
+Insert this action into your GitHub Actions workflow to upload your notebooks to
+your TileDB Cloud account. Generate a TileDB Cloud API token and save it as a
+GitHub repository secret, eg `TILEDB_CLOUD_TOKEN`
+
+Minimal example:
+
+```yaml
+    - uses: TileDB-Inc/github-actions/upload-notebooks@main
+      with:
+        notebooks: "notebook-1.ipynb notebook-2.ipynb notebook-3.ipynb"
+        TILEDB_CLOUD_TOKEN: ${{ secrets.TILEDB_CLOUD_TOKEN }}
+        TILEDB_CLOUD_NAMESPACE: <namespace>
+```
+
+Full example
+
+```yaml
+    - name: Upload notebooks
+      uses: TileDB-Inc/github-actions/upload-notebooks@main
+      with:
+        notebooks: "notebook-1.ipynb notebook-2.ipynb notebook-3.ipynb"
```

Comment:
It seems like it would be better to take this as an array rather than a single string.

Reply:
I agree. This is the first GitHub Action I have written, and this was the only format I could get to work

I also tried a YAML list:

```yaml 
        notebooks:
          - examples/OMETiff-convertor-demo.ipynb
          - examples/OMEZarr-convertor-demo.ipynb
 ```

And an array:

```yaml
       notebooks: ["examples/OMETiff-convertor-demo.ipynb", "examples/OMEZarr-convertor-demo.ipynb"]
```


https://github.com/jdblischak/TileDB-BioImaging/commit/3910cfbe7d34c6beb3f3684f95539c2d311c9781